apps/multiplatform/common/src/desktopMain/kotlin/chat/simplex/common/platform/ScrollableColumn.desktop.kt

```diff
@@ -135,9 +137,11 @@ actual fun LazyColumnWithScrollBarNoAppBar(
   // When scroll bar is dragging, there is no scroll event in nested scroll modifier. So, listen for changes on lazy column state
   // (only first visible row is useful because LazyColumn doesn't have absolute scroll position, only relative to row)
   val scrollBarDraggingState = remember { mutableStateOf(false) }
-  Box {
+  Box(contentAlignment = containerAlignment) {
```

Comment:
this component was no longer used, I did some changes to make it work as expected when we don't want bars to influence a simple lazy column with scrollbars on desktop